AggregateMetrics/AggregateMetrics/AzureWebApp/RateCounter.cs

```diff
@@ -1,21 +1,55 @@
-﻿using Microsoft.ApplicationInsights.DataContracts;
-using System;
-
-namespace Microsoft.ApplicationInsights.Extensibility.AggregateMetrics.AzureWebApp
+﻿namespace Microsoft.ApplicationInsights.Extensibility.AggregateMetrics.AzureWebApp
 {
+    using System;
+    using Microsoft.ApplicationInsights.DataContracts;
+    using Microsoft.ApplicationInsights.Extensibility.AggregateMetrics.Two;
+
     /// <summary>
     /// Struct for metrics dependant on time.
     /// </summary>
-    public struct RateCounter
+    public class RateCounter : ICounterValue
```

Comment:
Have "Gauge" in the name to be consistent with another `ICounterValue` implementation